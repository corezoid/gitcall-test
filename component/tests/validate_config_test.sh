#!/usr/bin/env bash
$CZ component generate-release ../ > "${CZ_TMP_DIR}/generated_release.yaml"
$CZ ops valid-config "${CZ_TMP_DIR}/generated_release.yaml" ../schema/main.json ../schema