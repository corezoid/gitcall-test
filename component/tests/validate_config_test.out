level=error msg="gitcall.config.dundergitcall.docker_daemon.host: Does not match format 'uri'"
level=error msg="gitcall.config.dundergitcall.docker_daemon.host: String length must be greater than or equal to 1"
level=error msg="gitcall.config.dundergitcall.global_policy.usercode_scale_prometheus_server: String length must be greater than or equal to 1"
level=error msg="gitcall.config.dundergitcall.global_policy: Must validate all the schemas (allOf)"
level=error msg="gitcall.config.dundergitcall.usercode_registry: String length must be greater than or equal to 1"
level=error msg="gitcall.config.dundergitcall.usercode_registry_schema: Invalid type. Expected: string, given: null"
level=error msg="gitcall.config.servicespimp.access_token: Invalid type. Expected: string, given: null"
level=error msg="gitcall.config.servicespimp.uri: Invalid type. Expected: string, given: null"
